src/main.rs

```diff
@@ -55,15 +57,25 @@ fn run() -> Result<(), Box<std::error::Error>> {
         )
         .arg(Arg::from_usage("[pyargs] 'args for python'").multiple(true));
     #[cfg(feature = "flame-it")]
-    let app = app.arg(
-        Arg::with_name("profile_output")
-            .long("profile-output")
-            .takes_value(true)
-            .help(
-                "the file to output the profile graph to. present due to being \
-                 built with feature 'flame-it'",
-            ),
-    );
+    let app = app
+        .arg(
+            Arg::with_name("profile_output")
```

Comment:
Next to the profile output, could we have an option enabling / disabling tracing at runtime in addition to the feature at compile time?

Reply:
I'm not sure, it'd require some sort of override of the `flame` dependency, and `flame-it` really shouldn't be enabled for a real release build, just developement. We could just disable outputting the data, I guess?

Reply:
No, what I mean is that if the flame-it feature is enabled, you get flame trace options at the command line. One of these options is then wether or not you want to use tracing.

I fully agree that trace should be disabled in release build for end users. It's mostly for ourselves to detect slowness of rustpython.